page-objects/pages/TASKListPage.js

```diff
@@ -16,13 +16,14 @@ const taskTitleContainer = '.assignment.card .title';
 const taskDescriptionContainer = '.assignment .text-muted.ckcontent';
 const taskContainer = '.homework li.card';
 const deleteTaskButtonInPopup = '.delete-modal button.btn-submit';
-const clickWithoutDueDate = "//*[text()='Task19']";
-const ungradedTask = "//div[text()='Task19']";
+const clickWithoutDueDate = "//div[@data-testid = 'taskTitle' and text() = 'Task19']";
```

Comment:
This will again look for the specific Task19. So either the variable should be called clickTask19, which wouldn't be good. Better would be: const withoutDueDateSection = "[data-testid='upperTaskSection']"; <- not a propper data-testid btw...